src/User32/User32+DISPLAY_DEVICE.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAY_DEVICE"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Receives information about the display device specified by the <c>iDevNum</c> parameter of the <see cref="User32.EnumDisplayDevices"/> function.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        public struct DISPLAY_DEVICE
+        {
+            /// <summary>
+            /// Size, in bytes, of the DISPLAY_DEVICE structure. This must be initialized prior to calling <see cref="User32.EnumDisplayDevices"/>.
+            /// </summary>
+            public uint cb;
+
+            /// <summary>
+            /// An array of characters identifying the device name. This is either the adapter device or the monitor device.
+            /// </summary>
+            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 32)]
+            public string DeviceName;
+
+            /// <summary>
+            /// An array of characters containing the device context string. This is either a description of the display adapter or of the display monitor.
+            /// </summary>
+            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 128)]
+            public string DeviceString;
+
+            /// <summary>
+            /// Device state flags. It can be any reasonable combination of the following.
```

Comment:
Drop the misleading teaser: " It can be any reasonable combination of the following." since you don't specify them, and on .NET where we have flags enums, it's unnecessary anyway.